ui.apps/src/main/content/jcr_root/apps/venia/clientlibs/common/js/CommerceGraphqlApi.js

```diff
@@ -46,19 +45,22 @@
          */
         async getProductImageUrls(productData) {
             //ugly but effective
-            let names = Object.keys(productData).reduce((acc, name) => (acc += '\"' + name + '\",'), '');
+            let names = Object.keys(productData).reduce((acc, name) => (acc += '"' + name + '",'), '');
             if (names.length === 0) {
                 return {};
             }
-            const query = `query { products(filter: {name: {in: [${names.substring(0, names.length - 1)}]}}) { items { sku name ... on ConfigurableProduct { variants { product { sku media_gallery_entries { file } } } } } } }`;
+            const query = `query { products(filter: {name: {in: [${names.substring(
+                0,
```

Comment:
This really became unreadable ... I'm not sure if there anything we can do about it.

Reply:
I used `// prettier-ignore` and properly formatted the GraphQL query. This should be much better now :)